bin/coreos-install

```diff
@@ -334,6 +343,27 @@ if [[ -n "${CLOUDINIT}" ]] || [[ -n "${COPY_NET}" ]]; then
     umount "${WORKDIR}/rootfs"
 fi
 
+if [[ -n "${IGNITION}" ]]; then
+  # The OEM partition should be #3 but make no assumptions here!
+  # Also don't mount by label directly in case other devices conflict.
+  OEM_DEV=$(blkid -t "LABEL=OEM" -o device "${DEVICE}"*)
+
+  if [[ -z "${OEM_DEV}" ]]; then
+    echo "Unable to find new OEM partition on ${DEVICE}" >&2
+    exit 1
+  fi
+
+  mkdir -p "${WORKDIR}/oemfs"
+  mount "${OEM_DEV}" "${WORKDIR}/oemfs"
+  trap "umount '${WORKDIR}/oemfs' && rm -rf '${WORKDIR}'" EXIT
+
+  echo "Installing Ignition config ${IGNITION}..."
+  cp "${IGNITION}" "${WORKDIR}/oemfs/${IGNITION}"
```

Comment:
$IGNITION may be an arbitrary path so it cannot be used as a target without trimming Amy directory components. May be better to just pick a name like ignition.json but that's up to you.